solidity/test/lib/deploy.ts

```diff
@@ -0,0 +1,53 @@
+// set this to turn off paramters checking in the deployment scripts
+process.env.TEST_DEPLOY_SCRIPTS = 'true';
+
+import { deployFungible as deployFungibleUpgradeable, deployNonFungible as deployNonFungibleUpgradeable } from '../../scripts/deploy_upgradeable';
+import { deployFungible as deployFungibleCloneable, deployNonFungible as deployNonFungibleCloneable } from '../../scripts/deploy_cloneable';
+import fungibilities from '../../scripts/tokens.json';
+import { ethers } from 'hardhat';
+
+export async function deployZeto(tokenName: string) {
+  let zeto, erc20, deployer;
+
+  let isFungible = false;
+  const fungibility = (fungibilities as any)[tokenName];
+  if (fungibility === 'fungible') {
+    isFungible = true;
+  }
+
+  if (process.env.USE_FACTORY !== 'true') {
+    // setup via the deployment scripts
```

Comment:
adding some logs here would be helpful to highlight how the contract was deployed through test run logs